### PR TITLE
Adds more translations of library/io

### PR DESCRIPTION
**Pull request by cschan1828**
Adds more translations of `library/io`

### CODE REVIEW

File: library/io.po [hunk 1]
```diff
@@ -302,6 +322,8 @@ msgid ""
 "This is a helper function for callables that use :func:`open` or :class:"
 "`TextIOWrapper` and have an ``encoding=None`` parameter."
 msgstr ""
+"這是個輔助函數，適用於使用 :func:`open` 或 :class:`TextIOWrapper` 且具有 "
+"``encoding=None`` 參數的 callables。"
```
Comment (mattwang44) [inline library/io.po]: nit: `callable` can be translated to `可呼叫物件` as it's in the glossary.
https://docs.python.org/zh-tw/3/glossary.html#term-callable

File: library/io.po [hunk 2]
```diff
@@ -39,6 +39,11 @@ msgid ""
 "any of these categories is called a :term:`file object`.  Other common terms "
 "are *stream* and *file-like object*."
 msgstr ""
+":mod:`io` 模組替 Python 提供處理各種類型 IO 的主要工具。有三種主要的 IO 類"
+"型： *文字 I/O (text I/O)*、*二進位 I/O (binary I/O)* 以及 *原始 I/O (raw I/"
```
Comment (mattwang44) [inline library/io.po]:
```suggestion
"型： *文字 I/O (text I/O)*、*二進位 I/O (binary I/O)* 以及\\ *原始 I/O (raw I/"
```

File: library/io.po [hunk 3]
```diff
@@ -39,6 +39,11 @@ msgid ""
...
+"O)*。這些均為泛用 (generic) 類型，且每種類型都可以使用各式後端儲存 (backing "
+"store)。任一種屬於這些類型的具體物件稱為 :term:`file object`。其它常見的名詞"
+"還有 *資料串流 (stream)* 以及 *類檔案物件 (file-like objects)*。"
```
Comment (mattwang44) [inline library/io.po]:
```suggestion
"還有\\ *資料串流 (stream)* 以及\\ *類檔案物件 (file-like objects)*。"
```

File: library/io.po [hunk 4]
```diff
@@ -116,6 +127,10 @@ msgid ""
 "be used for all kinds of non-text data, and also when manual control over "
 "the handling of text data is desired."
 msgstr ""
+"二進位 I/O（也稱為 *緩衝 I/O (buffered I/O)*）要求的是 :term:`bytes-like "
```
Comment (mattwang44) [inline library/io.po]:
```suggestion
"二進位 I/O（也稱為\\ *緩衝 I/O (buffered I/O)*）要求的是 :term:`bytes-like "
```

File: library/io.po [hunk 5]
```diff
@@ -116,6 +127,10 @@ msgid ""
...
+"objects <bytes-like object>` 且產生 :class:`bytes` 物件。不進行編碼、解碼或者"
```
Comment (mattwang44) [inline library/io.po]: nit: https://docs.python.org/zh-tw/3/glossary.html#term-bytes-like-object

```suggestion
"二進位 I/O（也稱為 *緩衝 I/O (buffered I/O)*）要求的是\\ :term:`類位元組"
"物件 (bytes-like object) <bytes-like object>` 且產生 :class:`bytes` 物件。不進行編碼、解碼或者"
```

File: library/io.po [hunk 6]
```diff
@@ -160,6 +175,11 @@ msgid ""
 "manipulate a raw stream from user code.  Nevertheless, you can create a raw "
 "stream by opening a file in binary mode with buffering disabled::"
 msgstr ""
+"原始 I/O（也稱為 *無緩衝 I/O (unbuffered I/O)*）通常作為二進位以及文字資料串"
```
Comment (mattwang44) [inline library/io.po]:
```suggestion
"原始 I/O（也稱為\\ *無緩衝 I/O (unbuffered I/O)*）通常作為二進位以及文字資料串"
```

File: library/io.po [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2001-2022, Python Software Foundation
+s Copyright (C) 2001-2022, Python Software Foundation
```
Comment (mattwang44) [inline library/io.po]: 這應該是不小心改到的？

Comment (cschan1828) [reply library/io.po]: Yeah..Thanks for catching this.

**Comment by cschan1828** [comment]
@mattwang44 Thanks for your review! I've revised according to your feedbacks.

**Review by mattwang44 — APPROVED** [review]
Thanks for the efforts!

**Pull request merged**